perses/app/relative_point_mutation_setup.py

```diff
@@ -238,11 +247,27 @@ def __init__(self,
                                                 molecules=molecules,
                                                 cache=None)
 
-        # Solvate apo and complex...
-        apo_input = list(self._solvate(protein_topology, protein_positions, water_model, phase, ionic_strength, apo_box_dimensions))
-        inputs = [apo_input]
+        # Solvate apo and complex (if necessary) and generate systems...
+        inputs = []
+        topology_list = [protein_topology]
+        positions_list = [protein_positions]
+        box_dimensions_list = [apo_box_dimensions]
         if ligand_input:
-            inputs.append(self._solvate(complex_topology, complex_positions, water_model, phase, ionic_strength, complex_box_dimensions))
+            topology_list.append(complex_topology)
+            positions_list.append(complex_positions)
+            box_dimensions_list.append(complex_box_dimensions)
+
+        for topology, positions, box_dimensions in zip(topology_list, positions_list, box_dimensions_list):
+            if solvate:
+                solvated_topology, solvated_positions = self._solvate(topology, positions, water_model, phase,
+                                                                      ionic_strength, box_dimensions)
+            else:
+                solvated_topology = topology
+                solvated_positions = unit.quantity.Quantity(value=np.array(
```

Comment:
there might be an issue here associated with the `solvate` function giving a tuple of posits rather than a list, but we'll see what the test says

Reply:
the tests pass, so i don't think there is any problem with this.